.github/workflows/e2e-conformance-trigger.yaml

```diff
@@ -7,6 +7,7 @@ jobs:
   conformance:
     if: github.repository == 'aws/karpenter' || github.event_name == 'workflow_dispatch'
     strategy:
+      max-parallel: 1
```

Comment:
Will this cause timeouts?

Reply:
It should not timeout. By default, each conformance job will have 6 hours to complete. The workflow has 35 days before timeout.

Reply:
Ok, sounds good. I'd be more apt to increase our API limits first before restricting this parallelism. 1 is going to mean that all of these jobs are going to take a really long time to run so I think we should do a little validation to tweak the value to something that makes sense after we get our limits/quotas bumped.